src/app/content/usage/usage-list/usage-list.component.html

```diff
@@ -2,21 +2,17 @@
     <hc-tab tabTitle="Term List" tabindex="2">
         <div>
             <div class="usage-table-filters">
-                <hc-form-field class="usage-table-filters-input" inline>
-                    <hc-label>Filter:</hc-label>
-                    <input hcInput (keyup)="applyFilter()" [formControl]="searchControl" placeholder="Search" />
-                    <hc-icon hcSuffix fontSet="fa" fontIcon="fa-times" (click)="resetSearch()"></hc-icon>
-                </hc-form-field>
+                <hc-search-bar [placeholder]="placeholder" [showSearchIcon]="showSearchIcon" [showClearIcon]="showClearIcon" (triggerSearch)="applyFilter($event)"></hc-search-bar>
```

Comment:
You could actually simplify this even further. THe example on cashmere is illustrating all of the inputs, but most of them default to what you are setting them to. You could remove all the properties you added in typescript and just have this in your template:

```
<hc-search-bar (triggerSearch)="applyFilter($event)"></hc-search-bar>
```

Reply:
Oh cool! Thanks for pointing that out @corykon I will give it a try!